msgcntr/messageforums-app/src/webapp/jsp/discussionForum/statistics/dfStatisticsList.jsp

```diff
@@ -35,8 +35,9 @@
 			  </div>
           </h:panelGroup>
           <h:panelGroup styleClass="itemNav specialLink">
-			<h:outputText value="#{msgs.cdfm_statistics} #{msgs.stat_byUser} " />
-			<h:outputText value="#{msgs.cdfm_toolbar_separator} " />
+			<h:outputText value="#{msgs.cdfm_statistics} #{msgs.stat_byUser}" style="padding-right: 5px;"/>
```

Comment:
Thanks for removing the nbsp. 

We have a $standard-space variable in the SASS files in Library that you can use for spaces.